src/adcs_io.c

```diff
@@ -59,6 +77,21 @@ ADCS_returnState request_uart_telemetry(uint8_t TM_ID, uint8_t* telemetry,
   return ADCS_OK;
 }
 
-ADCS_returnState request_i2c_telemetry(uint8_t* command, uint32_t length) {
+ADCS_returnState request_i2c_telemetry(uint8_t TM_ID, uint8_t *telemetry,
+                                       uint32_t length)
+{
+  uint8_t reply[length];
```

Comment:
I guess, we need some processing on the packets again, but if we didn't, do we really need the parameter reply or can we directly use telemetry?

Reply:
This is the same as the case for i2c_send_telecommand(). I was hoping i2c_receive to handle the processing for us using the HalCoGen drivers.

Reply:
Also, I guess we could directly use telemetry here, no need for reply.